sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/firestore/it/FirestoreTestingHelper.java

```diff
@@ -136,7 +136,7 @@ public FirestoreTestingHelper(CleanupMode cleanupMode) {
             .setCredentials(gcpOptions.getGcpCredential())
             .setProjectId(
                 firstNonNull(firestoreBeamOptions.getFirestoreProject(), gcpOptions.getProject()))
-            .setDatabaseId(firestoreBeamOptions.getFirestoreDb())
+            .setDatabaseId(firstNonNull(databaseId, firestoreBeamOptions.getFirestoreDb()))
```

Comment:
this will make the IT test always run against a database 'firestoredb' and you will not be able to override the database name from command line argument (e.g. --firestoreDb='myTestDb' which could be convenient when you run the test against your test database), you shouldn't need to hard code the IT test to set to 'firestoredb', it's configured in the gradle file, e.g.: https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/build.gradle#L194